po/manifest2po

```diff
@@ -92,7 +92,7 @@ function process_menu(menu) {
         if (menu[m].label) {
             push({
                 msgid: menu[m].label,
-                locations: [ filename ]
+                locations: [ filename + ":0" ]
```

Comment:
This should get a comment, as it's not obvious what that :0 does.

Reply:
Done